api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -396,6 +396,16 @@ message Cluster {
     google.protobuf.UInt64Value maximum_ring_size = 4 [(validate.rules).uint64 = {lte: 8388608}];
   }
 
+  // Specific configuration for the :ref:`Maglev<arch_overview_load_balancing_types_maglev>`
+  // load balancing policy.
+  message MaglevLbConfig {
+    // The table size for Maglev hashing. In fact, Maglev aims for ‘minimal disruption’ rather than an absolute guarantee.
+    // The minimal disruption means when the set of upstreams changes, a connection will likely be sent to the same
+    // upstream as it was before. So, The larger table size reduces disruption of hashing.
```

Comment:
maybe `Increasing the table size reduces the amount of disruption.` instead of the `So, the` phrasing?